brave/brave-core
# Migrates user settings to preferences

Resolves https://github.com/brave/brave-browser/issues/7024
Resolves https://github.com/brave/brave-browser/issues/7101
Resolves https://github.com/brave/brave-browser/issues/9547

DB schema change here https://github.com/brave/rewards/pull/8

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- start from master
- enable rewards
- change values for AC settings
- claim grant and do some tips
- close browser
- switch to this PR
- start browser again
- make sure that no data was lost (AC settings and summary report)

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
